src/avatar/Avatar.js

```diff
@@ -42,29 +44,12 @@ const Avatar = props => {
     imageProps,
     ...attributes
   } = props;
+  let { size } = props;
 
-  let { width, height } = props;
+  const iconDimension = typeof size === 'number' ? size : DEFAULT_SIZES[size];
```

Comment:
You forgot the change from @xavier-villelegier. This will still fail if I pass in `size="bob""`

Reply:
oh yeh sorry about that. I was thinking propTypes will fail it 
`PropTypes.Oneof(['small', '....'])

Reply:
Yea but propTypes is just a warning, it doesn't stop the app from trying to load. If we don't provide a fallback it'll red screen (crash)

Reply:
that is done